src/main/java/org/jsoup/nodes/Document.java

```diff
@@ -274,7 +428,6 @@ public Charset charset() {
          * @return the document's output settings, for chaining
          */
         public OutputSettings charset(Charset charset) {
```

Comment:
Maybe I'm missing it; how does changing this update the charset element? I.e. if I change the output settings after the parse (and normalize) is completed.